k8s/scripts/inspect.sh

```diff
@@ -62,6 +62,24 @@ function collect_sbom {
   cp --no-preserve=mode,ownership /snap/k8s/current/bom.json "$INSPECT_DUMP"/sbom.json
 }
 
+function collect_system_info {
+  mkdir -p $INSPECT_DUMP/sys
+  log_info "Copy processes list to the final report tarball"
+  ps -ef > $INSPECT_DUMP/sys/ps
+
+  log_info "Copy disk usage information to the final report tarball"
+  df -h | grep "^/" &> $INSPECT_DUMP/sys/disk_usage
+
+  log_info "Copy memory usage information to the final report tarball"
+  free -m > $INSPECT_DUMP/sys/memory_usage
+
+  log_info "Copy node uptime to the final report tarball"
+  uptime > $INSPECT_DUMP/sys/uptime
+
+  log_info "Copy /etc/os-release to the final report tarball"
+  cp /etc/os-release $INSPECT_DUMP/sys/etc-os-release
+}
```

Comment:
Let's also add the loaded kernel modules(smod), CPU info(lscpu), network connections (`ss -tuln`), mounts (`cat /proc/mounts`) and swap (`swapon`) - you never know.